src/git.js

```diff
@@ -254,9 +254,11 @@
     */
    Git.prototype.checkoutLatestTag = function (then) {
       var git = this;
-      return this.pull().tags(function (err, tags) {
-         git.checkout(tags.latest, then);
-      });
+      return this.pull(
+         () => git.tags(
```

Comment:
Please can this continue to the ES5 `function` form to allow for backward compatibility until the next major release of `simple-git`